### DISCUSSION TITLE
Brainstorming on future New Contributor or Intern projects

### DISCUSSION THREAD

**Discussion by ChristopherA** [BlockchainCommons/Community#34] [Ideas]
A discussion of different ideas for new contributors or internship projects for our community. Don't limit them to technical and coding, they could include law, documentation, translating, etc.

**Comment by ChristopherA** [comment]
I'd love to see something like a distributed or decentralized [CryptPad](https://github.com/xwiki-labs/cryptpad) behind a TorGap.

**Comment by ChristopherA** [comment]
Add fee estimation to SpotBit. 

See https://twitter.com/ChristopherA/status/1354205776118640642?s=20 and https://bitcoindevkit.org/blog/2021/01/fee-estimation-for-light-clients-part-1/

/cc @RCasatta

**Comment by namcios** [comment]
Some ideas on activism and HRF use cases:
- leverage Smart Custody book's chapter on Risk Modeling and gear it to activists, for example documentation on the creation of a new pseudonymous identity while considering threats etc

**Comment by namcios** [comment]
Another one:
- Thorough documentation on self-sovereignty tools on raspberry pis for example: a VPN/Tor guide, a secure cold storage setup with ColdCard connected to own Bitcoin node, BTCPayServer instance for donations, and self-hosting other tools such as BitWarden for password management and Standard Notes for encrypted notes (for example with EmbassyOS and/or Yunohost), also CoinJoining stack with Samourai Dojo + Samourai Wallet for private mixing and spending

**Comment by namcios** [comment]
Also something else which I've been thinking about for a while: how to move from using tools that hamper one's privacy to tools that help and empower it (for example, from Gmail to ProtonMail/Tutanota, from usual Android to De-Googled Android such as Graphene or Calyx, from Google Chrome to De-Googled Chromium or Firefox with adjusted settings for better security and privacy, etc)

Don't know how much of an improvement those moves would actually be. It's just the sense I get from [informal] research on the topic.
